tests/test_setget_field.f90

```diff
@@ -58,13 +54,19 @@ program test_setget_field
 #endif
   
   fld => backend%allocator%get_block(DIR_X, VERT)
+  fld_c => backend%allocator%get_block(DIR_C, VERT)
+  allocate(arr, mold=fld_c%data)
```

Comment:
This will fail on the CUDA backend. We should't access a `field`s data member unless we're certain that it's data stored on the host. You can instead use `field%get_shape` to get back a length 3 integer array with the correct dimensions.

Reply:
Done